scipy/stats/tests/test_qmc.py

```diff
@@ -593,42 +586,39 @@ def test_sample_stratified(self, optimization, centered, strength):
                 assert_equal(res_set, desired)
 
     def test_discrepancy_hierarchy(self):
-        seed = np.random.RandomState(123456)
+        seed = 68756348576543
         lhs = qmc.LatinHypercube(d=2, seed=seed)
         sample_ref = lhs.random(n=20)
         disc_ref = qmc.discrepancy(sample_ref)
 
-        seed = np.random.RandomState(123456)
         optimal_ = qmc.LatinHypercube(d=2, seed=seed, optimization="random-CD")
         sample_ = optimal_.random(n=20)
         disc_ = qmc.discrepancy(sample_)
 
         assert disc_ < disc_ref
 
     def test_raises(self):
-        seed = np.random.RandomState(12345)
-
         message = r"'toto' is not a valid optimization method"
         with pytest.raises(ValueError, match=message):
-            qmc.LatinHypercube(1, seed=seed, optimization="toto")
+            qmc.LatinHypercube(1, optimization="toto")
```

Comment:
And here?

Reply:
Here it does not really matter as we check for invalid inputs.

Reply:
OK, another case of it not being needed before.